src/py42/clients/securitydata.py

```diff
@@ -307,8 +296,35 @@ def _get_other_file_location_version(self, md5_hash, sha256_hash):
             version = self._preservation_data_service.find_file_version(
                 md5_hash, sha256_hash, paths
             )
-            if version.status_code != 204:
-                return version
+            if version.status_code == 200:
+                return version.data
+
+    def _get_file_stream(self, version):
+        if version.get(u"edsUrl"):
+            return self._get_exfiltrated_file(version)
+
+        return self._get_stored_file(version)
+
+    def _get_stored_file(self, version):
```

Comment:
Could swap the placement of `_get_exfiltrated_file()` and `_get_stored_file()` to match the order of usage in the method above.